common4j/src/main/com/microsoft/identity/common/java/util/IPlatformUtil.java

```diff
@@ -129,4 +131,11 @@ public interface IPlatformUtil {
      */
     @Nullable
     List<Map.Entry<String, String>> updateWithAndGetPlatformSpecificExtraQueryParameters(@Nullable List<Map.Entry<String, String>> originalList);
+
+    /**
+     * Store telemetry region by tenant.
+     *
+     * @param clientInfo the client info containing tenant information and tdbr claim
+     */
+    void storeTelemetryRegionByTenant(@NonNull IStorageSupplier supplier, @NonNull ClientInfo clientInfo);
```

Comment:
I honestly think there is no reason for this method to in the platform util. This method can easily be in its own class or any class related to telemetry helpers. and it would still give you platform specific flexibility because you are passing a generic `IStorageSupplier` to the params which takes care of that already (so no need to do double platform agnostic-ness by putting it on platform util)

Reply:
Done